syft/cataloger/python/package_cataloger.go

```diff
@@ -18,6 +18,12 @@ const (
 	wheelMetadataGlob = "**/*dist-info/METADATA"
 )
 
+type pythonPackageData struct {
```

Comment:
This piece of data seems to be referred to as an "entry" ubiquitously in the code — it may be worthwhile to rename the type to include "entry" for clarity as it's being used